js/main.js

```diff
@@ -202,7 +202,8 @@ function startNeverEndingGame (images) {
 
         if (signal.settings) {
           settings = {
-            duration: (signal.settings.duration && signal.settings.duration.value) || defaultSettings.duration
+            duration: (signal.settings.duration && signal.settings.duration.value) || defaultSettings.duration,
```

Comment:
`defaultSettings` has been removed so I guess this could crash?

Reply:
It does not currently crash because `signal.settings` is given, but it’s not safe. Thanks!